src/pages/log-management/opensearch/opensearch-dashboards/api.mdx

```diff
@@ -84,4 +81,6 @@ Both curl requests call the same API Endpoint. This request returns detailed inf
 resource usage, the server status, installed plugins, plugin statuses and metrics.
 
 There are many other Opensearch Dashboards features that can be accessed via REST API through the Logit.io Proxy. These features are explained in the 
-Elastic docs here [here](https://opensearch.org/docs/latest/api-reference/)
\ No newline at end of file
+Elastic docs here [here](https://opensearch.org/docs/latest/api-reference/)
```

Comment:
Needs both Elastic and Opensearch links

Reply:
@leesmith110 Just to check, it needs both for the article under opensearch dashboards? I was going to put the elastic docs as a link in an article under Kibana